backend/src/main/kotlin/com/woowahan/campus/zzimkkong/domain/DayOfWeeks.kt

```diff
@@ -18,5 +18,11 @@ enum class DayOfWeeks {
                     valueOf(day)
                 }
         }
+
+        fun create(weekday: String): DayOfWeeks {
+            return values()
+                .firstOrNull { it.name == weekday.uppercase() }
+                ?: throw IllegalArgumentException("유효하지 않은 요일입니다.")
+        }
```

Comment:
```suggestion
        fun create(weekday: String): DayOfWeeks {
            return requireNotNull(values().firstOrNull { it.name == weekday.uppercase() }) { "유효하지 않은 요일입니다." }
        }
```

`IllegalArgumentException` 직접 throw 하는 방법 말고 이렇게 `requireNotNull` 사용해도 될 것 같은데 이건 어떤가요??
사실 아직 어떤 방식이 더 코틀린스러운진 모르겠네요 🤣